### ISSUE TITLE
Utilisation de la police Fontawesome

### ISSUE THREAD

**Issue by dumbbell** [AuxQuatreCoinsDeParis/aux-quatre-coins-de-paris.fr#3]
Visiblement, la poice Fontawesome met des plombes à charger. Où en a-t-on besoin ? Est-ce vraiment utile ? Peut-on améliorer son chargement ?

**Comment by dumbbell** [comment]
Une piste intéressante : utiliser la version SVG de la police et les inclure dans l'HTML au moment du rendu côté serveur par Hugo.
https://www.client9.com/using-font-awesome-icons-in-hugo/

**Comment by Aurelielb** [comment]
le problème de la version svg embeddée, c'est que ce sera beaucoup moins
facile à utiliser pour Audrey. Il faudra faire plus de shortcodes custom ou
alors elle devra les inclure elle même, ça va devenir un vrai bordel.
Par contre je peux alléger le tout en retirant les icônes qu'on utiliserai
jamais: dans ce cas il faudra faire une liste des icônes restantes, est-ce
que ça vous irai ?

**Comment by dumbbell** [comment]
Je suis d'accord, c'est plus compliqué pour Audrey, ce qui peut-être un problème. Faut que je lui montre ce que l'article propose.

En fait, je me base sur la rapport de GTMetrix.com (temps pour une connexion 3G émulée), mais je n'arrive pas à uploaded le screenshot ici (GitHub me retourne une erreur)... en gros, y'a le chargement de la home qui prend 1,23" puis la CSS de Fontawesome qui met 1,12" de plus.

Voici le lien vers le rapport, je ne sais pas si tu peux le consulter :
https://gtmetrix.com/reports/aux-quatre-coins-de-paris.fr/j0lrxctU

**Comment by Aurelielb** [comment]
Yep mais on voit aussi ce genre de chose tout simplement en faisant un
audit lighthouse sous chrome ^^

**Comment by Aurelielb** [comment]
Ah j'ai répondu un peu vite, c'est pas tout à fait ce que je croyais en
fait, du coup ça semble un peu plus précis à ce niveau
